packages/core/upload/server/controllers/admin-folder-file.js

```diff
@@ -32,7 +32,20 @@ module.exports = {
     const folderService = getService('folder');
 
     const deletedFiles = await fileService.deleteByIds(body.fileIds);
-    const deletedFolders = await folderService.deleteByIds(body.folderIds);
+    const {
+      folders: deletedFolders,
+      totalFolderNumber,
+      totalFileNumber,
+    } = await folderService.deleteByIds(body.folderIds);
+
+    if (deletedFiles.length + deletedFolders.length > 1) {
```

Comment:
Shouldn't this be `>= 1`?

Reply:
No because otherwise we don't know if the user clicked on the bulk button or not (because this route is also used when you delete a folder through the edit view of the folder), which is the thing we want to track